test/extended/networking/ipsec.go

```diff
@@ -483,6 +698,94 @@ var _ = g.Describe("[sig-network][Feature:IPsec]", g.Ordered, func() {
 			waitForIPsecConfigToComplete(oc, v1.IPsecModeDisabled)
 			checkForGeneveOnlyTraffic(config)
 		})
+
+		// This test sets up IPsec mode to External, chooses two worker nodes as left and right
+		// servers for IPsec configuration, generate and import certs into their nss db, configure
+		// its ipsec conf via nmstate and then check node traffic between them are IPsec encrypted.
+		g.It("validate node traffic is IPsec encrypted for corresponding IPsec north south configuration [apigroup:config.openshift.io] [Suite:openshift/network/ipsec]", func() {
```

Comment:
Don't we need to test this in `IPsecModeFull` as well?

Reply:
This would create IPsec mesh connectivity on all of the cluster nodes. hence just enabling `IPsecModeExternal` and choosing two cluster nodes for IPsec via NS configuration and testing pod traffic.

Reply:
Makes sense. But this is actually a testing gap, isn't it? Maybe add a comment acknowledging it here.

Reply:
yes, this is because of unavailability of external node to test NS IPsec even with `IPsecModeFull` mode. will add a comment about this.

Reply:
done